samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -247,6 +250,14 @@ static const char link_rsrp_usage_str[] =
 	"  -s, --subscribe,    Subscribe for RSRP info\n"
 	"  -u, --unsubscribe,  Unsubscribe for RSRP info\n";
 
+static const char link_search_usage_str[] =
+	"Usage: link search --read | --write | --start | --clear\n"
+	"Options:\n"
+	"  -r, --read,       Read modem search configuration\n"
+	"      --write,      Write modem search configuration\n"
+	"      --start,      Start modem search request\n"
+	"      --clear,      Clear modem search configuration\n";
```

Comment:
Would it be more logical to have "read", "write" and "clear" first because those three affect the search configuration, while "start" is a different operation and starts an extra search? Maybe you could also add the word "extra" into the "start" option description, because normally periodic searches are done automatically in limited service, based on the given configuration.

The command description also doesn't mention that it's for "periodic" searches, not sure if this should also be visible.

Reply:
Descriptions will need a lot more text once I'm ready with the implementation of the writing part which is the most complex to implement. I'll see which parts of the text will be copied from AT command descriptions.

Reply:
Now it's a bit more complete and let's see what the audience says.